modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -3,12 +3,29 @@
 using namespace cv;
 
 unsigned countCoins(const Mat& img) {
-    Mat gray, thresh;
-    cvtColor(img, gray, COLOR_BGR2GRAY);
```

Comment:
Please check yours notebook's settings or `tabs` and `spaces`, because you created redundant differences depend with different count spaces. Please correct this.